sdk/quoting/src/config.ts

```diff
@@ -0,0 +1,49 @@
+import { Currency } from '@uniswap/sdk-core'
+import { FeeAmount } from '@uniswap/v3-sdk'
+import { USDC_TOKEN, WETH_TOKEN } from './libs/constants'
+
+// Sets if the example should run locally or on chain
+export enum Environment {
+  LOCAL,
+  WALLET_EXTENSION,
```

Comment:
Do we need a wallet version of this one since we don't actually connect / use the concept of a wallet?